app/ts/background/simulationModeHanders.ts

```diff
@@ -168,7 +169,7 @@ export async function sendRawTransaction(
 async function singleCallWithFromOverride(ethereumClientService: EthereumClientService, simulationState: SimulationState, request: EthCallParams, from: bigint) {
 	const callParams = request.params[0]
 	const blockTag = request.params.length > 1 ? request.params[1] : 'latest' as const
-	const input = callParams.data !== undefined ? callParams.data : new Uint8Array()
+	const inputAndDataMerged = 'data' in callParams && callParams.data !== undefined ? callParams.data : 'input' in callParams && callParams.input !== undefined ? callParams.input : new Uint8Array()
```

Comment:
It may be worth a helper since this is needed in multiple places.
```suggestion
	const inputAndDataMerged = 'data' in callParams && callParams.data !== undefined
		? callParams.data
		: 'input' in callParams && callParams.input !== undefined
			? callParams.input
			: new Uint8Array()
```